src/app/dao/tests/PartDAO.spec.ts

```diff
@@ -60,43 +84,363 @@ describe('PartDAO', () => {
-            await createConnectedGoogleUser(false);
+            creator = await createConnectedUser(CREATOR_EMAIL, CREATOR_NAME);
         });
         it('should return true when user has an active part as player zero', async() => {
             // Given a part where user is player zero
-            await dao.create({ ...part, playerZero: username });
+            await partDAO.create({ ...part, playerZero: creator });
             // When checking if the user has an active part
-            const result: boolean = await dao.userHasActivePart(username);
+            const result: boolean = await partDAO.userHasActivePart(creator);
             // Then it should return true
             expect(result).toBeTrue();
         });
         it('should return true when user has an active part as player one', async() => {
             // Given a part where user is player zero
-            await dao.create({ ...part, playerOne: username });
+            await partDAO.create({ ...part, playerOne: creator });
             // When checking if the user has an active part
-            const result: boolean = await dao.userHasActivePart(username);
+            const result: boolean = await partDAO.userHasActivePart(creator);
             // Then it should return true
             expect(result).toBeTrue();
         });
         it('should return false when the user has no active part', async() => {
-            // Given a part where the user is not active
-            await dao.create(part);
+            // Given no part where the user is active
             // When checking if the user has an active part
-            const result: boolean = await dao.userHasActivePart(username);
+            const result: boolean = await partDAO.userHasActivePart(creator);
             // Then it should return false
             expect(result).toBeFalse();
 
         });
         afterEach(async() => {
-            await signOut(TestBed.inject(Auth));
+            await signOut();
+        });
+    });
+    describe('for verified user', () => {
+        it('should allow creating a part', async() => {
+            // Given a verified user
+            const creator: MinimalUser = await createConnectedUser(CREATOR_EMAIL, CREATOR_NAME);
+            // When creating a part
+            const result: Promise<string> = partDAO.create({ ...PartMocks.INITIAL, playerZero: creator });
+            // Then it should succeed
+            await expectAsync(result).toBeResolved();
+        });
+        it('should allow reading parts', async() => {
+            // Given a part and a verified user
+            const creator: MinimalUser = await createConnectedUser(CREATOR_EMAIL, CREATOR_NAME);
+            const part: Part = { ...PartMocks.INITIAL, playerZero: creator };
+            const partId: string = await partDAO.create(part);
+            await signOut();
+
+            await createConnectedUser(MALICIOUS_EMAIL, MALICIOUS_NAME);
```

Comment:
it's not malicious then :p !

Reply:
il peut l'être néanmoins :D Mais mieux avec candidate oui, changé